packages/rollup-plugin/.gitignore

```diff
@@ -0,0 +1 @@
+dist
```

Comment:
Optional: 
Hmmm WDYT about adding something like `packages/*/dist` into the root .gitignore instead?

Reply:
More risky/opaque as to why something is "gitignored". I'll leave it for now.